### PR TITLE
scroll token

### PR DESCRIPTION
**Pull request by Jibz-Mysten**
## Description 

Describe the changes or additions included in this PR.

Scroll 

https://user-images.githubusercontent.com/126525197/234365779-22f9b258-5c3e-4ecd-9d1a-1fb33f58e4c7.mov

### CODE REVIEW

**Comment by mystie711** [comment]
lol. did i ask for this feature?
omg. Sorry.
Let's discard this work please.

**Comment by mystie711** [comment]
> lol. did i ask for this feature? omg. Sorry. Let's discard this work please.

spoke with Godwin and realized what the actual intent was. 
We are going to make the whole page scrollable when there are many coins.

**Review by mystie711 — APPROVED** [review]

File: apps/wallet/src/ui/app/pages/home/tokens/TokensDetails.tsx [hunk 1]
```diff
@@ -48,9 +48,9 @@ function MyTokens() {
             {balance?.length ? (
                 <div className="flex flex-1 justify-start flex-col w-full mt-6">
                     <Text variant="caption" color="steel" weight="semibold">
-                        MY COINS
+                        My Coins
                     </Text>
-                    <div className="flex flex-col w-full justify-center divide-y divide-solid divide-steel/20 divide-x-0 px-1.5">
+                    <div className="flex flex-col w-full justify-start divide-y divide-solid divide-steel/20 divide-x-0 px-1.5 pb-1">
```
Comment (williamrobertson13) [inline apps/wallet/src/ui/app/pages/home/tokens/TokensDetails.tsx:53]: Hmm, I don't see how this fixes the issue - changing the justification doesn't do anything (I don't think the `div` container needs to be flexed in the first place) and 4px of bottom padding doesn't seem to do much either. The wallet doesn't seem to be scrollable if I have three coins in my wallet (pic 1), and if I have five coins then the fifth coin is hidden behind the tab container (pic 2). LMK if I'm misunderstanding something 😄 

image

image

**Review by williamrobertson13 — CHANGES_REQUESTED** [review]

File: apps/wallet/src/ui/app/pages/home/tokens/TokensDetails.tsx (hunk 1 above)
Comment (Jibz-Mysten) [reply apps/wallet/src/ui/app/pages/home/tokens/TokensDetails.tsx:53]: my bad, should not have change it

File: apps/wallet/src/ui/app/pages/home/tokens/TokensDetails.tsx [hunk 2]
```diff
@@ -48,9 +48,9 @@ function MyTokens() {
             {balance?.length ? (
                 <div className="flex flex-1 justify-start flex-col w-full mt-6">
                     <Text variant="caption" color="steel" weight="semibold">
-                        MY COINS
+                        My Coins
                     </Text>
-                    <div className="flex flex-col w-full justify-center divide-y divide-solid divide-steel/20 divide-x-0 px-1.5">
+                    <div className="flex flex-col w-full justify-center divide-y divide-solid divide-steel/20 divide-x-0 px-1.5 pb-20">
```
Comment (Jibz-Mysten) [inline apps/wallet/src/ui/app/pages/home/tokens/TokensDetails.tsx:53]: adding a large padding-bottom to offset the menu

File: apps/wallet/src/ui/app/pages/home/tokens/TokensDetails.tsx [hunk 3]
```diff
@@ -48,9 +48,9 @@ function MyTokens() {
             {balance?.length ? (
                 <div className="flex flex-1 justify-start flex-col w-full mt-6">
                     <Text variant="caption" color="steel" weight="semibold">
-                        MY COINS
+                        My Coins
                     </Text>
-                    <div className="flex flex-col w-full justify-center divide-y divide-solid divide-steel/20 divide-x-0 px-1.5">
+                    <div className="flex flex-col w-full justify-center divide-y divide-solid divide-steel/20 divide-x-0 px-1.5 mb-20">
```
Comment (Jibz-Mysten) [inline apps/wallet/src/ui/app/pages/home/tokens/TokensDetails.tsx:53]: adding margin-bottom to offset the menu

**Review by williamrobertson13 — APPROVED** [review]

**Pull request merged**